services/web/server/src/simcore_service_webserver/login/_constants.py

```diff
@@ -24,6 +24,15 @@
 MSG_PASSWORD_CHANGED: Final[str] = "Your password is changed"
 MSG_PASSWORD_MISMATCH: Final[str] = "Password and confirmation do not match"
 MSG_PHONE_MISSING: Final[str] = "No phone was registered for this user"
+MSG_UNAUTHORIZED_CODE_RESEND_2FA: Final[
+    str
+] = "For security reasons, you cannot resend 2FA code anymore, please restart."
```

Comment:
Maybe it is ok to replace "For security reasons" with "Unauthorised" in these messages